Sources/Tokenizer.swift

```diff
@@ -10,6 +10,18 @@ extension String {
     var singleQuoteCount = 0
     var doubleQuoteCount = 0
 
+    let specialCharacters = ",|:"
+    func appendWord(_ word: String) {
+      if components.count > 0 &&
+        (specialCharacters.characters.contains(components.last!.characters.last!) ||
+          specialCharacters.contains(word))
+      {
+        components[components.count-1] += word
+      } else {
+        components.append(word)
+      }
+    }
```

Comment:
Wouldn't it be more logical to use a `Set<Character>` here which would seem more appropriate and already handle all that append-without-duplicates logic for you already?

Reply:
That's not exactly what we are doing here. We are checking if a word is a special character (for the case when it is surrounded by spaces, i.e. in ` | `), or if previous word ends with a special character, in which case we concatenate last and new word (so `ars| default` ends up as `ars|default`).

Reply:
Ah right gotcha